packages/admin/admin-rte/src/core/Controls/LinkControls.tsx

```diff
@@ -52,6 +62,10 @@ export default (p: IControlProps) => {
 };
 
 declare module "@mui/material/styles" {
+    interface ComponentsPropsList {
+        CometAdminMyComponent: RteLinkControlsProps;
+    }
```

Comment:
Please name the key correctly:

```suggestion
    interface ComponentsPropsList {
        CometAdminRteLinkControls: RteLinkControlsProps;
    }
```

And if you add the Props to `ComponentsPropsList` please also add them to `defaultProps` in the `Components` interface below:

```ts
defaultProps?: Partial<ComponentsPropsList["CometAdminRteLinkControls"]>;
```